tools/cndump/globals.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright (C) 2022 Micron Technology, Inc.  All rights reserved.
+ */
+
+#ifndef CNDUMP_GLOBALS_H
+#define CNDUMP_GLOBALS_H
+
+#include <stdbool.h>
+
+struct global_opts {
+    bool help;
+    bool verbose;
+};
+
+extern struct global_opts global_opts;
+extern const char *progname;
```

Comment:
It would be nice to have a header like this with in libhse-cli where there was a .c file that had a function that took the argv[0] and setup the program name `extern const char *progname`. We do that in all our tools in pretty much the same way.